### ISSUE TITLE
WakaTime Erorr Atom-Nightly

### ISSUE THREAD

**Issue by MashdorDev** [wakatime/wakatime-cli#432]
Hey, I'm hoping you are doing well

I'm using Atom-Nightly **V 1.59.0**
 and I'm getting an error from the plugin while I use it 

_here is the console log -_

 ```
logger.js:47 [WakaTime] [DEBUG] [wakatime.js:285] Current wakatime-cli version is 
logger.js:47 [WakaTime] [DEBUG] [wakatime.js:286] Checking for updates to wakatime-cli...
logger.js:47 [WakaTime] [DEBUG] [wakatime.js:330] GitHub API Response 304
logger.js:47 [WakaTime] [DEBUG] [wakatime.js:293] Found an updated wakatime-cli v1.7.4
logger.js:47 [WakaTime] [DEBUG] [wakatime.js:402] Downloading wakatime-cli from https://github.com/wakatime/wakatime-cli/releases/download/v1.7.4/wakatime-cli-windows-amd64.zip
logger.js:47 [WakaTime] [DEBUG] [wakatime.js:420] Extracting wakatime-cli.zip file...
logger.js:47 [WakaTime] [DEBUG] [wakatime.js:108] Finished installing wakatime-cli.
logger.js:47 [WakaTime] [DEBUG] [wakatime.js:182] Finished initializing WakaTime.
logger.js:47 [WakaTime] [DEBUG] [wakatime.js:583] C:\Users\Dor\.wakatime\wakatime-cli-windows-amd64.exe --entity D:\Dor\Studies\GA\React-Portfolio\src\index.css --plugin atom-wakatime/11.0.4 --lineno 9 --verbose --config C:\Users\Dor\.wakatime.cfg --alternate-project React-Portfolio
logger.js:51 [WakaTime] [WARN] [wakatime.js:621] Error: Command failed: C:\Users\Dor\.wakatime\wakatime-cli-windows-amd64.exe --entity D:\Dor\Studies\GA\React-Portfolio\src\index.css --plugin atom-wakatime/11.0.4 --lineno 9 --verbose --config C:\Users\Dor\.wakatime.cfg --alternate-project React-Portfolio

**Comment by MashdorDev** [comment]
My ``wakatime.log``

```INFO 2021/05/31 11:58:15 Attempting to read in config file
DEBUG 2021/05/31 11:58:15 Reading file:  C:\Users\Dor\.wakatime.cfg
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/cmd/legacy/run.go:103","func":"Run","level":"debug","message":"command: heartbeat","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/cmd/legacy/heartbeat/heartbeat.go:65","func":"SendHeartbeats","level":"debug","message":"heartbeat params: category: 'coding', cursor position: '', entity: 'D:\\Dor\\Studies\\GA\\React-Portfolio\\src\\index.css', entity type: 'file', num extra heartbeats: 0, hostname: 'DESKTOP-1IQTUEI', is write: false, language: '', line number: '9', lines in file: '', offline disabled: false, offline sync max: 100, time: 1622476695.76077, api params: (api key: 'bdf0...', api url: 'https://api.wakatime.com/api', plugin: 'atom-wakatime/11.0.4', timeout: 1m0s), filter params: (exclude: '[]', exclude unknown project: false, include: '[]', include only with project file: false), network params: (disable ssl verify: false, proxy url: '', ssl cert filepath: ''), project params: (alternate: 'React-Portfolio', disable submodule: '[]', map patterns: '[]', override: ''), sanitize params: (hide branch names: '[]', hide file names: '[]', hide project names: '[]')","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/filter/filter.go:28","func":"1","level":"debug","message":"execute heartbeat filtering","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/filestats/filestats.go:28","func":"1","level":"debug","message":"execute filestats detection","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/language/language.go:20","func":"1","level":"debug","message":"execute language detection","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/deps/deps.go:34","func":"1","level":"debug","message":"execute dependency detection","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/deps/deps.go:116","func":"Detect","level":"debug","message":"No parser has been found for language \"CSS\". Using Unknown parser to detect dependencies.","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/project/project.go:60","func":"1","level":"debug","message":"execute project detection","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/heartbeat/sanitize.go:27","func":"1","level":"debug","message":"execute heartbeat sanitization","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/offline/offline.go:51","func":"1","level":"debug","message":"execute offline queue","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/offline/offline.go:81","func":"1","level":"debug","message":"include 37 heartbeat(s) from offline queue","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/api/heartbeat.go:26","func":"Send","level":"debug","message":"sending 38 heartbeat(s) to api at https://api.wakatime.com/api/v1/users/current/heartbeats.bulk","now":"2021-05-31T11:58:15-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/offline/offline.go:87","func":"1","level":"debug","message":"api error: failed making request to \"https://api.wakatime.com/api/v1/users/current/heartbeats.bulk\": Post \"https://api.wakatime.com/api/v1/users/current/heartbeats.bulk\": x509: certificate signed by unknown authority","now":"2021-05-31T11:58:16-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/pkg/offline/offline.go:88","func":"1","level":"debug","message":"pushing 38 heartbeat(s) to offline queue","now":"2021-05-31T11:58:16-04:00","version":""}
{"caller":"/home/runner/work/wakatime-cli/wakatime-cli/cmd/legacy/heartbeat/heartbeat.go:47","func":"Run","level":"fatal","message":"failed to send heartbeat(s): failed to send heartbeats via api client: failed making request to \"https://api.wakatime.com/api/v1/users/current/heartbeats.bulk\": Post \"https://api.wakatime.com/api/v1/users/current/heartbeats.bulk\": x509: certificate signed by unknown authority","now":"2021-05-31T11:58:16-04:00","version":""}

**Comment by alanhamlett** [comment]
For now, you can add this line to your `~/.wakatime.cfg` file to fix it:

    no_ssl_verify = true

Later, it will be fixed along with #385.

**Comment by bruteforks** [comment]
was getting the same errors because I accidentally left debug=true - in case anyone else has a similar looking problem
